src/modules/core/components/Fields/Select/Select.css

```diff
@@ -156,3 +189,22 @@
 .baseSelect.sizeLarge {
   font-size: var(--size-large);
 }
+
+.activeOptionColorBlue {
```

Comment:
We would prefer that you don't modify our core components and just use 2 options instead:
- target the change with the css from the component where you use the core component
- create a wrapper component around where you would add the necessary changes.

The second option is least preferable and should be used only when there are several places where it's used and the code would be repeated.

In this case I feel that the active option color can easily be done via css without adding any extra props so please remove and do it via css.